documentation/Using-Azure-TestFramework.md

```diff
@@ -76,7 +76,7 @@ To use this option, set the following environment variable before starting Visua
 `tenantId`: The [AAD Tenant ID](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-tenant-id)
 
 
-	TEST_CSM_ORGID_AUTHENTICATION=SubscriptionId={SubId};ServicePrincipal={clientId};ServicePrincipalSecret={clientSecret};AADTenant={tenantId};Environment={env};HttpRecorderMode=Record;
+	TEST_CSM_ORGID_AUTHENTICATION=SubscriptionId={SubId};ServicePrincipal={clientId};Password={clientSecret};AADTenant={tenantId};Environment={env};HttpRecorderMode=Record;
```

Comment:
I was prompted to make this change because using ServicePrincipalSecret did not work for me. It would still prompt me to enter in my credentials in an AAD popup.

What is the "new Framework" you are referring to? How do I ensure that our tests are using it?